### DISCUSSION TITLE
DepotDownloader closes instantly upon startup

### DISCUSSION THREAD

**Discussion by Ralkero** [SteamRE/DepotDownloader#509] [Q&A]
I have installed .NET Framework 8, but DepotDownloader simply closes upon opening. I don't see any errors in the CMD window, so I have no idea what is causing this. 

I am on Windows 11 Pro 23H2, 64 bit.

**Comment by voided** [comment]
See https://github.com/SteamRE/DepotDownloader/discussions/440#discussioncomment-6246671
